@navikt/core/react/src/form/combobox/SelectedOptions/selectedOptionsContext.tsx

```diff
@@ -12,11 +12,14 @@ import { ComboboxProps } from "../types";
 
 type SelectedOptionsContextType = {
   addSelectedOption: (option: string) => void;
+  canSelectMoreOptions: boolean;
   isMultiSelect?: boolean;
   removeSelectedOption: (option: string) => void;
   prevSelectedOptions?: string[];
   selectedOptions: string[];
+  maxSelectedOptions?: number;
```

Comment:
maxSelectedOptions -> numMaxSelected

Check if we can combine addNewValue, numMaxSelected++ in an option object

Reply:
maxSelected: {
  isLimitReached,
  limit,
  message
}

Reply:
Har implementert objekt-strukturen over.